src/lc0ctl/onnx2leela.cc

```diff
@@ -230,6 +236,77 @@ bool ValidateNetwork(const pblczero::Net& weights) {
   return true;
 }
 
+const std::vector<float> rule50weights = {
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1.0f / 99, 1, 1};
+
+bool FixRule50(pblczero::ModelProto& model, const std::string& in) {
+  bool flag = false;
+
+  std::string name = "rule50fix";
+
+  for (size_t i = 0; i < model.graph().node_size(); i++) {
+    auto node = model.graph().node(i);
+    for (size_t j = 0; j < node.input_size(); j++) {
+      if (node.input(j) == in) {
+        CERR << "Inerting scaling between " << in << " and " << node.name();
+        model.mutable_graph()->mutable_node(i)->mutable_input()->at(j) =
+            std::string(name);
+        flag = true;
+      }
+    }
+  }
+
```

Comment:
Isn't it an error if we reach this place and the flag is false?
(also the variable name `flag` here and in other places, if still needed, should probably be replaced with better names)

Reply:
Done, this was a remnant from an older iteration.